### DISCUSSION TITLE
V18: Chips Component removed

### DISCUSSION THREAD

**Discussion by britesoft** [primefaces/community#2775] [PrimeNG]
You have remove Chips component and asked to use Autocomplete with "multiple = true and typeahead=false. But it is not the exact replacement. Chips had some unique features. Unless you add those features to Autocomplete, it is unfortunate removal of the component

**Comment by qwadrox** [comment]
@britesoft Hi. You're not alone with this insight: https://github.com/primefaces/primeng/issues/16390 
For now, you can follow this issue and support with a thumbs up.
